.github/workflows/deploy-alpha.yml

```diff
@@ -0,0 +1,33 @@
+name: deploy-alpha
+
+concurrency: alpha
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  deployment:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    environment: alpha
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Setup Node
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Dependencies
```

Comment:
Why we don't use lerna bootsrap here, like in the test CI

Reply:
the bootstrap command is in the postinstall script ;)

Reply:
Good :)